movie/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,115 @@
+import { BrowserRouter as Router } from "react-router-dom";
+import { FaSearch, FaPlus } from "react-icons/fa";
```

Comment:
NOTED, For later. Your can use MUI-icon: https://mui.com/material-ui/material-icons/

So we don't need an extra package just for icon